src/test/java/ArrayStackTest.java

```diff
@@ -22,7 +22,7 @@ void createStack() {
     class WhenNewEmpty {
 
         @Test
-        void push_successfulAdd_returnsTrue() {
+        void push_many_returnsTrue() {
```

Comment:
Presumably these are supported to be `empty` and such.

Reply:
idk how this happened, but OK.